version/src/main/java/org/jboss/as/version/ProductConfig.java

```diff
@@ -89,6 +91,10 @@ private ProductConfig(ModuleLoader loader, ProductConfProps productConfProps, Ma
                     productVersion = manifest.getMainAttributes().getValue("JBoss-Product-Release-Version");
                     consoleSlot = manifest.getMainAttributes().getValue("JBoss-Product-Console-Slot");
                     projectName = manifest.getMainAttributes().getValue("JBoss-Project-Release-Name");
+                    projectBanner = manifest.getMainAttributes().getValue("JBoss-Project-Banner");
+                    if(projectBanner != null) {
+                        projectBanner = projectBanner.replaceAll("\\\\n", System.lineSeparator());
```

Comment:
Since it looks like those manifest entries always process through a logger you could avoid this by using `%n` in the manifest.

Reply:
ok

Reply:
This won't work since it's an argument. Sorry about that confusion.